source/_posts/2018-01-31-infrastrucutre-as-code-a-beginners-perspective.markdown

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title: "Infrastructure as Code: A beginner's perspective"
+category: infrastructure
+comments: true
+author:
+  name: Federico Kauffman
+  email: federico@wyeworks.com
+  twitter_handle: fedekauffman
+  github_handle: fedekau
+  image: /images/team/federico-kauffman.jpg
+  description: Software Engineer at WyeWorks. Currently working with Javascript and Ruby. Learnaholic.
+---
+
+A couple of months ago I came to realize that DevOps might be something I like, I was not sure what to do, so I talked to the VP of Engineering at WyeWorks, after that, we found a way for me to experiment and work in a DevOps position for several months.
```

Comment:
I think this would read better as multiple sentences instead of commas.
- after 'something I like'       <-- something >that< I like ?
- after 'VP of Engineering at WyeWorks'